### PR TITLE
Update wordpress packages (again)

### PR DESCRIPTION
**Pull request by noahtallen**
While working on #78711, some new versions came out. So this PR actually updates these packages to the latest version. Hopefully we can get Renovate running soon to automate this again!

## To do:

see what CI comes up with!

## Testing instructions:
- CI should pass
- Smoke test Calypso and the major workflows

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -62,25 +62,34 @@ packageExtensions:
   "@signal-noise/stylelint-scales@2.0.3":
     peerDependencies:
       postcss: "*"
-  "@wordpress/commands@0.6.2":
+  "@wordpress/block-editor@12.5.0":
```
Comment (noahtallen) [inline .yarnrc.yml]: Unfortunately, it seems like these peer dependency issues are regressing. About two years ago, I got everything fixed internal to Gutenberg, but seems like that's stopped happening again, leaving these issues scattered about everywhere.

File: package.json [hunk 2]
```diff
@@ -297,91 +297,91 @@
...
-		"@wordpress/block-serialization-default-parser": "4.36.0",
-		"@wordpress/block-serialization-spec-parser": "4.36.0",
-		"@wordpress/blocks": "12.12.0",
-		"@wordpress/browserslist-config": "5.19.0",
-		"@wordpress/components": "25.1.0",
-		"@wordpress/compose": "6.12.0",
-		"@wordpress/core-data": "6.12.0",
-		"@wordpress/create-block-tutorial-template": "2.24.0",
-		"@wordpress/create-block": "4.20.0",
+		"@wordpress/a11y": "3.37.0",
```
Comment (noahtallen) [inline package.json]: I used our new script to update all these, and it worked great

**Review by tyxla — COMMENTED** [review]
Seems like we might have to accommodate the changes from #79675.

**Comment by noahtallen** [comment]
Rebased!

**Review by tyxla — APPROVED** [review]
I've tested a few signup and checkout flows, as well as some logged-in and logged-out Calypso pages, then compared the devdocs WP components pages for desert. 

Everything looks good and safe to go 🚀 

Thanks @noahtallen, exciting to see us finally reach the destination  🙌

**Comment by noahtallen** [comment]
I'll deploy tomorrow, though you don't need to wait for me!

**Pull request merged**